dataframe/CMakeLists.txt

```
@@ -2,28 +2,32 @@
 file(GLOB sources *.cc)
 file(GLOB headers *.h)
 
-PODIO_GENERATE_DICTIONARY(libedm4hepRDF edm4hepRDF.h SELECTION selection.xml USES edm4hep)
 
-add_library(edm4hepRDF SHARED ${sources} ${headers} libedm4hepRDF.cxx )
-target_link_libraries(edm4hepRDF edm4hep podio::podio ROOT::Physics)
+add_library(edm4hepRDF SHARED ${sources} ${headers})
+target_link_libraries(edm4hepRDF edm4hep ROOT::Physics)
+PODIO_GENERATE_DICTIONARY(edm4hepRDF ${headers} SELECTION selection.xml USES edm4hepRDF
+  OPTIONS --library edm4hepRDF${CMAKE_SHARED_LIBRARY_SUFFIX}
```

@vvolkl infact this was wrong, and makes the install step fail
```suggestion
  OPTIONS --library libedm4hepRDF${CMAKE_SHARED_LIBRARY_SUFFIX}
```

---

Ah. I was just looking into why the `pcm` files for edm4hep and dataframe were named differently.